### ISSUE TITLE
iOS13 Simulator Problem as in #437

### ISSUE THREAD

**Issue by TRCarbone** [kishikawakatsumi/KeychainAccess#462]
I'm also having the same problem with the Touch ID enabled keychain item in the iOS13 simulator.

I noted that the issue was closed, but is there any resolution to this?

**Comment by kishikawakatsumi** [comment]
@TRCarbone All tests passed in my environment. Probably it seems you have not properly configured keychain entitlements. To further investigate, I need more info. What tests failed? What Xcode version are you using? etc.

**Comment by TRCarbone** [comment]
@kishikawakatsumi I'm on Xcode 11.2.1. I'm running iPhone 8 Simulator with iOS 13.2.2 where it fails. On my physical iPhone 7 with 13.2.3 it works just fine.

I'd e happy to provide test results, but being a bit new at this I'm struggling with how to use it. I believe I need to start a clean single app project and set it somehow as the target.

Any pointers on this would be appreciated!

**Comment by kishikawakatsumi** [comment]
@TRCarbone Thanks for details. Let me make sure do the failures happen only on the simulator tests? I mean, no error happens on physical iOS 13 devices with an app that was built Xcode 11? If so, it doesn't seem a fatal problem.

**Comment by TRCarbone** [comment]
@kishikawakatsumi Correct... it is fine on a physical device but not the simulator. Not "fatal" but it makes ongoing development harder as I test most everything on the simulator as I code.

It may be a bug in the simulator, but it would be nice if there were a workaround.Can you try the tests on the simulator?

**Comment by kishikawakatsumi** [comment]
> but it makes ongoing development harder 

Absolutely. I have no idea about any workaround or fix the problem yet, but I'll investigate it soon. Thanks.

**Comment by TRCarbone** [comment]
@kishikawakatsumi Just to keep you posted, it is still failing after updating to Xcode 11.3 and the 13.3 simulator.
